lib/resources/r4/document_reference.yaml

```
@@ -29,3 +94,140 @@ fields:
     - display: US Core DiagnosticReport Category
       system: http://loinc.org
       info_link: http://hl7.org/fhir/us/core/ValueSet/us-core-diagnosticreport-category
+
+- name: subject
+  required: 'Yes'
+  type: Reference
+  action:
+    - create
+  description: Who or what the document is about.
```

```suggestion
  description: Who the document is about.
```

Since we're only supporting Patient references, we can just keep it as 'who'.

---

You can find the changes here: https://github.com/cerner/fhir.cerner.com/pull/446/commits/eb5ca8d7a6905720c70e0c203d9de99739180394